acprep

```diff
@@ -535,140 +535,46 @@ class PrepareBuild(CommandLineApp):
-                            'libedit-dev',
-                            'texinfo',
-                            'lcov',
-                            'libutfcpp-dev',
-                            'sloccount'
-                        ])
-                    elif release == 'xenial':
-                        packages.extend([
-                            'doxygen',
-                            'cmake',
-                            'ninja-build',
-                            'zlib1g-dev',
-                            'libbz2-dev',
-                            'python-dev',
-                            'libgmp3-dev',
-                            'libmpfr-dev',
-                            'gettext',
-                            'libedit-dev',
-                            'texinfo',
-                            'lcov',
-                            'libutfcpp-dev',
-                            'sloccount'
-                        ])
-                    elif release == 'saucy':
-                        packages.extend([
-                            'doxygen',
-                            'cmake',
-                            'ninja-build',
-                            'zlib1g-dev',
-                            'libbz2-dev',
-                            'python-dev',
-                            'libgmp-dev',
-                            'libmpfr-dev',
-                            'gettext',
-                            'libedit-dev',
-                            'texinfo',
-                            'lcov',
-                            'sloccount'
-                        ])
-                    elif release == 'precise':
-                        packages.extend([
-                            'libtool',
-                            'cmake',
-                            'zlib1g-dev',
-                            'libbz2-dev',
-                            'python-dev',
-                            'libgmp-dev',
-                            'libmpfr-dev',
-                            'gettext',
-                            'libedit-dev',
-                            'texinfo',
-                            'lcov',
-                            'libutfcpp-dev',
-                            'sloccount'
-                        ])
+                    release_list = [
+                        "noble",   # 24.04
+                        "jammy",   # 22.04
+                        "focal",   # 20.04
+                        "eoan",    # 19.10
+                        "bionic",  # 18.04
+                        "xenial",  # 16.04
+                        "trusty",  # 14.04
+                        "saucy",   # 13.10
+                        "precise", # 12.04
+                    ]
+                    if release in release_list:
+                        # Common Ubuntu packages
+                        packages.extend(
+                            [
+                                "cmake",
+                                "libgmp-dev" "libmpfr-dev",
```

Comment:
Is this missing a comma here?
```suggestion
                                "libgmp-dev", "libmpfr-dev",
```

Reply:
Yes, fixed. Thanks for pointing that out!